securedrop_client/storage.py

```diff
@@ -208,12 +208,13 @@ def __update_submissions(model: Union[Type[File], Type[Message]],
                                 if s.uuid == submission.uuid][0]
 
             # Update files on disk to match new filename.
-            if (local_submission.filename != submission.filename):
-                rename_file(data_dir, local_submission.filename,
-                            submission.filename)
+            if (
+                (local_submission.is_downloaded and not local_submission.is_decrypted) and
+                (local_submission.filename != submission.filename)
+            ):
+                rename_file(data_dir, local_submission, submission.filename)
```

Comment:
could you make this entire if block easier to read please, or update the comment above since we're no longer simple just updating the files on disk to match the server filenames